travel-event-host-app/src/lib/category.ts

```diff
@@ -0,0 +1,11 @@
+enum Category {
+  travelAndOutdoor = 'Travel and Outdoor',
+  socialActivities = 'Social Activities',
+  hobbiesAndPassions = 'Hobbies and Passions',
+  sportAndFitness = 'Sport and Fitness',
+  healthAndWellbeing = 'Health and Wellbeing',
+  technology = 'Technology',
+  artAndCulture = 'Art and Culture',
+  games = 'Games',
+}
```

Comment:
Let's update it so it's something like this. The camelCase text will be what is saved in the database. 
```suggestion
  TravelAndOutdoor = 'travelAndOutdoor',
  SocialActivities = 'socialActivities',
  ...
}
```